sdk/identity/test-resources-post.ps1

```diff
@@ -66,9 +66,9 @@ mvn -ntp clean install -DskipTests "-Drevapi.skip=true" "-Dcheckstyle.skip=true"
 
 Write-Host "Fetching a new federated token for Azure Resource Manager..."
 $ARM_OIDC_TOKEN = az account get-access-token --resource "https://management.azure.com/" --query accessToken -o tsv
-Write-Host "##vso[task.setvariable variable=ARM_OIDC_TOKEN]$ARM_OIDC_TOKEN"
-Write-Host "##vso[task.setvariable variable=ARM_OIDC_TOKEN]$ARM_OIDC_TOKEN"
-$env:ARM_OIDC_TOKEN = $ARM_OIDC_TOKEN
+
+# Set ARM_OIDC_TOKEN as a global pipeline variable for all subsequent steps
```

Comment:
What are you attempting to do here? We need to be careful about output a secret and also looks like this might clash with the existing token we are writing.